### ISSUE TITLE
docker path incorrect for sdk

### ISSUE THREAD

**Issue by tlodge** [me-box/databox#276]
mounting /sdk rather than [databoxroot]/sdk so not finding correct settings.json

**Comment by Toshbrown** [comment]
Hi @tlodge 

You can test this fix now no need to wait for a new version ;-) 

```
docker pull databoxsystems/databox:latest
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $(pwd)/certs:/certs -v $(pwd)/sdk:/sdk -v -t databoxsystems/databox:latest /databox sdk -start --host-path $(pwd) -release latest
```

I've also added the env vars  DATABOX_VERSION and DATABOX_DEFAULT_REGISTRY


@tlodge if you lloked at this befor this edit the cmd above was wrong, sorry

**Comment by Toshbrown** [comment]
fix in latest, will be in 0.5.1
